src/display/text_layer.js

```diff
@@ -540,55 +561,53 @@ var renderTextLayer = (function renderTextLayerClosure() {
         this._expanded = true;
         this._bounds.length = 0;
       }
-      if (expandDivs) {
-        for (var i = 0, ii = this._textDivs.length; i < ii; i++) {
-          var div = this._textDivs[i];
-          var transform;
-          var width = div.dataset.originalWidth;
-          if (div.dataset.canvasWidth !== undefined && width > 0) {
-            // Dataset values are of type string.
-            var textScale = div.dataset.canvasWidth / width;
+
+      for (var i = 0, ii = this._textDivs.length; i < ii; i++) {
+        var div = this._textDivs[i];
+        var divProperties = this._textDivProperties.get(div);
+
+        if (expandDivs) {
+          var transform = '';
+          var width = divProperties.originalWidth;
```

Comment:
Nit: Since you're directly accessing `divProperties.canvasWidth` below, I think that you can probably do the same thing with `divProperties.originalWidth` as well and not use the temporary `width` variable.